serverless/README.md

```diff
@@ -59,25 +59,26 @@ Note: If you did not modify the provided `template.yml` file when you installed
+| `enableDDLogs`          | Enable Datadog log collection for the Lambda function. Note: This setting has no effect on logs sent via the Datadog Forwarder. Defaults to true.                                                                                                                                                                                                                                                                                                                                                   |
+| `service`               | When set, the macro adds a `service` tag to all Lambda functions with the provided value.                                                                                                                                                                                                                                                                                                                                                                                                           |
+| `env`                   | When set, the macro adds an `env` tag to all Lambda functions with the provided value.                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `logLevel`              | Sets the log level. Set to `DEBUG` for extended logging.                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| `captureLambdaPayload`  | Automatically tags a function request and response so it is displayed in the APM application.                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Would this be more accurate? "Automatically tags the function execution span with request and response payloads, so they can be displayed in the APM application."

Reply:
@astuyve , is `DD_CAPTURE_LAMBDA_PAYLOAD` documented in the public docs somewhere? I'm having trouble finding it. It might be a good idea to link to it from here.